### DISCUSSION TITLE
Task 06 - Problem 6 : Card Name Sequence Order!

### DISCUSSION THREAD

**Discussion by hastishwan-uw** [chulminy/CIVE497-CIVE700#169] [Q&A] [answered]
Do we need to show the before and after pics, just like the screenshots, if everything’s swapped out right? Or is it ok if we show the result after the swap?

![image](https://github.com/chulminy/CIVE497-CIVE700/assets/48458021/0bd64112-d15e-4d35-a01b-c6ba8e5082ff)

**Comment by chulminy** [comment, marked as answer]
Yes. it's correct!
